### PR TITLE
Feature/P2022-9

### PR DESCRIPTION
**Pull request by sayuuri**
Add Mapstruct configuration with example working Mapping Interface

### CODE REVIEW

**Review by Pawel-Koz — APPROVED** [review]

**Review by jasokolowska — APPROVED** [review]
Looks fine

File: pom.xml [hunk 1]
```diff
@@ -74,13 +82,29 @@
...
             </plugin>
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/*MapperImpl.class</exclude>
+                    </excludes>
+
```
Comment (eveento) [inline pom.xml:106]: remove empty space line

File: src/main/java/com/intive/patronage22eganortap/dto/UserDto.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.intive.patronage22eganortap.dto;
```
Comment (eveento) [inline src/main/java/com/intive/patronage22eganortap/dto/UserDto.java:1]: why we need this class?

File: src/main/java/com/intive/patronage22eganortap/mapper/GlobalMapperConfig.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.intive.patronage22eganortap.mapper;
```
Comment (eveento) [inline src/main/java/com/intive/patronage22eganortap/mapper/GlobalMapperConfig.java:1]: think about better structure for project for mappers

File: src/main/java/com/intive/patronage22eganortap/mapper/UserMapper.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package com.intive.patronage22eganortap.mapper;
```
Comment (eveento) [inline src/main/java/com/intive/patronage22eganortap/mapper/UserMapper.java:1]: why we need this mapper?

File: src/main/java/com/intive/patronage22eganortap/model/User.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package com.intive.patronage22eganortap.model;
```
Comment (eveento) [inline src/main/java/com/intive/patronage22eganortap/model/User.java:1]: the same

File: src/test/java/com/intive/patronage22eganortap/UserMapperTest.java [hunk 6]
```diff
@@ -0,0 +1,75 @@
+package com.intive.patronage22eganortap;
```
Comment (eveento) [inline src/test/java/com/intive/patronage22eganortap/UserMapperTest.java:1]: there is no point. these tests make non sense

Comment (eveento) [reply src/test/java/com/intive/patronage22eganortap/UserMapperTest.java:1]: what are you exactly try to test?

File: pom.xml [hunk 7]
```diff
@@ -74,13 +82,29 @@
...
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/*MapperImpl.class</exclude>
```
Comment (eveento) [inline pom.xml:104]: where is config? how to achieve it? i didn't catch it

**Comment by eveento** [comment]
not accepted

**Pull request merged**